app/components/onboarding_response/onboarding_response.html.erb

```diff
@@ -3,5 +3,6 @@
     <%= c 'icon', icon: icon, class: 'OnboardingResponse-icon' %>
     <%= c('heading', tag: :h1) { heading } %>
     <p><%= text %></p>
+    <%= content %>
```

Comment:
Why is this necessary?

Reply:
Ah, nevermind – it was necessary before, but became irrelevant with my changes! :)